var/spack/repos/builtin/packages/log4cxx/package.py

```diff
@@ -14,14 +14,21 @@ class Log4cxx(CMakePackage):
 
     maintainers = ['nicmcd']
 
+    version('0.12.1', '807058734cebf7c92e99484313becbc6be63e48bd930889c733318bb78dfec498b66aaa12255ed30879abb005b32e2b965f9dd87ec50d0421ca067f3f66280d9')
```

Comment:
nit: Can you match the 0.12.1 line to the 0.12.0 line?

Reply:
I just copied the sha512 has value from the download site. I guess I could replace it with the sha256 value. @sethrj is it too late for me to do that, I'm not sure when the auto-merge will occur.

Reply:
I disabled automerge, you can update to use the sha256 value.